acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubCreationPage.java

```diff
@@ -108,7 +108,8 @@ public void selectPipelineToCreate(String pipeline){
     }
 
     public void clickCreatePipelineButton() {
-        wait.until(ExpectedConditions.elementToBeClickable(createPipelineButton)).click();
+        wait.until(ExpectedConditions.elementToBeClickable(createPipelineButton));
+        wait.until(ExpectedConditions.visibilityOf(createPipelineButton)).click();
```

Comment:
Which error are you trying to defend against here?

Reply:
@cliffmeyers this was an alternate fix to what I included in #1484 the issue is the first click fails because the button is moving

Reply:
Yeah I don't think this code change will help, it could have used `WaitUtil.click` but to your point, we can just merge in #1484 instead.